### DISCUSSION TITLE
Importing CSS files add up to 17 seconds per page load

### DISCUSSION THREAD

**Discussion by bennett-sh** [carbon-design-system/carbon-components-svelte#1718] [Q&A]
I've currently included the following line in my __layout.svelte:
```typescript
import "carbon-components-svelte/css/g90.css"
```
The problem is that includes everything resulting in a very large file taking ~17 seconds to load each time you visit a page. Is there a way to only include the parts you need making it load quicker?

**Comment by theetrain** [comment]
Unfortunately, using these CSS files does include all component styles—as well as the components you're not using. The alternative is to import SCSS component files on an as-needed basis; which will result in a slightly slower developer experience in exchange for serving only what's needed to your users. This is an issue we hope to resolve by v1.0.0.

See "SCSS' styling option on the docs: https://carbon-components-svelte.onrender.com/

<img width="961" alt="image" src="https://user-images.githubusercontent.com/12798751/230738806-17d4b998-a934-4ad9-8a4f-067aec5bb020.png">
